rules-emerging-threats/2024/Malware/Raspberry-Robin/registry_set_internet_settings_zonemap.yml

```diff
@@ -0,0 +1,42 @@
+title: Potential Raspberry Robin Registry Set Internet Settings Zonemap
+id: 16a4c7b3-4681-49d0-8d58-3e9b796dcb43
+status: experimental
+description: Detecting registry modifications related to the proxy configuration of the system, potentially associated with the Raspberry Robin malware, as seen in campaigns running in Q1 2024.
```

Comment:
Can you please enhance the description? Why does Raspberry Robin enable these registry keys? Please elaborate.

Reply:
cc @swachchhanda000